tutorials/fundamentals/then-async.md

```diff
@@ -0,0 +1,33 @@
+When making API requests, you can either use `.then()` or `async/await`. Internally, they are the same thing.
+The difference is that in an `async/await` function, the entire function execution will be halted on the first instance of a promise whereas when using
```

Comment:
"the entire function execution will be halted on the first instance of a promise" -> "JavaScript will pause the function execution until the promise settles"